examples/region_sharding/lookup_vindex.json

```diff
@@ -0,0 +1,24 @@
+{
+  "sharded": true,
+  "vindexes": {
+    "customer_region_lookup": {
+      "type": "consistent_lookup_unique",
+      "params": {
+        "table": "main.customer_lookup",
```

Comment:
In the original example on https://vitess.io/docs/user-guides/region-sharding/ , we also had to define the primary vindex on the lookup table:

```
    "customer_lookup": {
      "column_vindexes": [
	{
            "column": "keyspace_id",
            "name": "identity"
        }
      ]
    },
```

Is that missing from this example? Or are you making the lookup table unsharded?

Is there another PR with the changes to the tutorial document https://vitess.io/docs/user-guides/region-sharding/ ?

I'm curious to see a discussion in the tutorial on the sharding strategy for the lookup table.

- If we shard by `id` (which makes the most sense from a functionality point-of-view; otherwise the lookup table is useless), or if we make the table unsharded, then we might still have to go cross-region for the lookups, which will add latency.
- Unless we cross-replicate every shard into every region, but then there are questions about replication lag consistency when using the lookup table.

Reply:
I'm using the new workflows that weren't available when we first built the example. `CreateLookupVindex` also creates a primary vindex on the lookup table.
I'm still working on rewriting the user guide, PR should be ready shortly.

Reply:
Do you think it is helpful for `CreateLookupVindex` to return the created vschema? Otherwise it is silent and not obvious.

Reply:
I haven't run vitess yet or gotten familiar with the `vtctlclient` commands, so I'm not in a good position to say. But I'll agree that it is not obvious, and finding ways to make it more obvious would probably be very helpful.